de-DE/lang/numere.nlng

```diff
@@ -121,6 +121,7 @@ PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICD_[NUM]=					ellipticD(x,k)
 PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICE_[NUM]=					ellipticE(x,k)                                              VAL      - Wert des elliptischen Integrals E(x,k) an der Stelle x mit 0 <= k < 1. %%DEFINED_REAL%%
 PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICF_[NUM]=					ellipticF(x,k)                                              VAL      - Wert des elliptischen Integrals F(x,k) an der Stelle x mit 0 <= k < 1. %%DEFINED_REAL%%
 PARSERFUNCS_LISTFUNC_FUNC_ELLIPTICPI_[NUM]=					ellipticPi(x,n,k)                                           VAL      - Wert des elliptischen Integrals Pi(x,n,k) an der Stelle x mit 0 <= k < 1. %%DEFINED_REAL%%
+PARSERFUNCS_LISTFUNC_FUNC_ENDSWITH_[STRING]=				endswith(sStr)                                              STR      - Gibt den Wert, True zur�ck, wenn der zweite Substring am Ende des Strings gefunden wurde, sonst False

```

Comment:
Misses the second argument, e.g. `endswith(sStr,sEnd)`. The function type is also `LOG` and not `STR`. That's valid for all function strings here